tools/const_finder/test/fixtures/lib/target.dart

```diff
@@ -13,3 +13,24 @@ class Target {
     print('$stringValue $intValue');
   }
 }
+
+class ExtendsTarget extends Target {
+  const ExtendsTarget(String stringValue, int intValue, Target targetValue)
+      : super(stringValue, intValue, targetValue);
+}
+
+class ImplementsTarget implements Target {
```

Comment:
Consider also adding a test where Target is mixed-in (e.g. `class Foo with Target`).

Reply:
Done .. I think.